uwmodem_simulation.orogen

```diff
@@ -4,16 +4,23 @@ import_types_from "uwmodem_simulationTypes.hpp"
 using_library "usbl_evologics"
 using_task_library 'usbl_evologics'
 
+# Instantiate one Task per way.
+# Connect inputs in transmitter and outputs in receiver.
 task_context "Task" do
     needs_configuration
 
-    # Set interface. SERIAL or ETHERNET.
+    # Set interface. SERIAL or ETHERNET. Default ETHERNET
     # This will affect the rate of transmission data in raw_data_output
     # For SERIAL it is 2 bytes/Rawpacket. For ETHERNET it is a direct transmission.
-    property "interface", "/usbl_evologics/InterfaceType"
+    property "receiver_interface", "/usbl_evologics/InterfaceType", :ETHERNET
     # Set average distance in meters between modems.
     # It affects the transfer delay. It does not affect the rate of lost data.
     property "distance", "double", 300
+    # Set probability of good transmission in %
+    # It will affect the lost of data
+    property "probability", "int", 90
```

Comment:
Percentage is a bit like degrees .... Don't use them. Use probabilities in [0, 1] (floating point)